### PR TITLE
 Support for config and state containers get, and changing key, table separators to :

### PR DESCRIPTION
**Pull request by justinejose91**

### CODE REVIEW

**Comment by justinejose91** [comment]
@anand-kumar-subramanian , Looks like key and table separators have got changed to | in your recent changes for subscribe. If we use | as table, key separator, it will break the counters get for interfaces. I guess it was a typo while cleanup. Please review it.

**Review by jeffyin-dell — APPROVED** [review]

**Comment by jeffyin-dell** [comment]
Will wait for @anand-kumar-subramanian approval prior to merge.
Edit: approved offline, merging..

**Pull request merged**
